x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/ConfigInitialNode.java

```diff
@@ -434,7 +434,7 @@ private GeneralNames getSubjectAltNames() throws IOException {
             }
         }
         // this is the unequivocal, non-standard, mark for a cert generated by this auto-config process
-        generalNameSet.add(new GeneralName(GeneralName.otherName, CertGenUtils.createCommonName(ConfigInitialNode.class.getName())));
+        generalNameSet.add(CertGenUtils.createCommonName(ConfigInitialNode.class.getName()));
```

Comment:
We should probably settle to use a static name like "Autogenerated CA" and we don't need to add this in the SANs, but only as the subject of the certificate. Will tackle as part of one of the follow-up PRs